Vagrantfile

```diff
@@ -34,25 +34,23 @@ Vagrant.configure(2) do |config|
     config.vm.box = "ubuntu/vivid64"
     ubuntu_common config
   end
-  config.vm.define "wheezy" do |config|
-    config.vm.box = "debian/wheezy64"
-    deb_common(config)
-  end
+  # Wheezy's backports don't contain Openjdk 8 and the backflips required to
+  # get the sun jdk on there just aren't worth it. We have jessie for testing
+  # debian and it works fine.
```

Comment:
I'm ok with dropping it because Wheezy is no longer Debian stable. And this is master which won't be released for a while.

I was playing around with it - I can get Oracle Java 8 on there if I spend some more time on it but I'd love to get this in so the master tests are sane again. We can add an issue for wheezy if we really want it back.